### DISCUSSION TITLE
Cyrillic file names

### DISCUSSION THREAD

**Discussion by nonameformr** [photoprism/photoprism#3032] [General]
Are you support Cyrillic file names?
I upload files from WebDAV.

![Screenshot_2022-12-24-15-12-25-489_com.android.chrome.jpg](https://user-images.githubusercontent.com/11997188/209435832-344a72ea-7223-415b-93f4-030cf86024c2.jpg)

**Comment by lastzero** [comment]
It supports Unicode. If that is 90s style legacy ISO encoded non ASCII text, it might not work aka look like this.

**Reply by nonameformr** [reply]
Thanks. Put the appropriate check mark in the Total Commander plugin
